gammapy/makers/utils.py

```diff
@@ -213,6 +292,8 @@ def make_map_background_irf(
         Background rate model.
     geom : `~gammapy.maps.WcsGeom`
         Reference geometry.
+    fov_rotation_step : `~astropy.units.Quantity`
+        Maximum rotation error to create sub-time bins if the irf is 3D and in AltAz.
```

Comment:
Do you test somewhere that this is a 3D IRF? What happens in the contrary?

Reply:
I test `if bkg.has_offset_axis or bkg.fov_alignment == FoVAlignment.RADEC`
which handle the 2D and 3D, RaDec aligned cases. 
Then line 333 `if bkg.fov_alignment == FoVAlignment.ALTAZ:`
handles the 3D AltAz case. Issues could arise if a 2D IRF has `(bkg.fov_alignment == FoVAlignment.ALTAZ) == True`

Looking at the current classes it should not happen randomly since the the default of fov_alignment is RaDec. But I  will add a layer of check. Using line 333 :
`if is_instance(bkg, Background3D) and bkg.fov_alignment == FoVAlignment.ALTAZ:`
or 
`if not bkg.has_offset_axis and bkg.fov_alignment == FoVAlignment.ALTAZ:`

This will allow to ignore the AltAz handling when the data irf was already evaluated as 2D.